pagopa/io-eudiw-app
# feat: [EUDIW-175] Add PID issuance flow

> [!WARNING]
> Depends on #46 and #47  

> [!NOTE]
> Tests will be added later

## Short description
This PR implements the PID issuance flow. It doesn't implement any wallet related login as showing a card or opening the details of the credentials.

## List of changes proposed in this pull request
- Add `Reactotron` middleware to debug the app;
- Update env files with the needed variables;
- Add `io-react-native-login-utils` to perform strong auth;
- Update `io-app-design-system` and a few other dependencies; 
- Add components for showing the UI carried over from the old implementation; 
- Add components to parse and verify claims with `zod`;
- Add saga for the PID issuance; 
- Add credentials reducer to store credentials;
- Add the settings screen which allows the reset of the wallet instance and also of the onboarding process for test purposes.

## How to test
Start the app and try to obtain a PID. After adding it to the wallet the banner in the home screen should disappear. You can test again the flow by resetting the wallet instance status from the settings.
